### ISSUE TITLE
Voting for skip/add time/restart should add your vote by default

### ISSUE THREAD

**Issue by AtomicLiquid** [EvoTM/EvoSC#125]
When you start a vote on a server using EvoSC, it's quite essential that you starting the vote will count as a vote per default. Seems sort of counterintuitive that you, as the vote starter, don't count in, but you have to add it after you've started the vote.

**Comment by snixtho** [comment]
I think this depends on a situation actually. For example as an admin, you may end up in the situation where you are not sure about what to do and would ask the people, especially for custom votes, you don't necessarily want your vote to influence the outcome.

**Comment by brakerb** [comment]
Yeah like snix mentioned, there are situations where you want to be neutral in the vote.
But we could add an option in user settings, where they could select whether or not to automatically vote when starting one.

**Comment by snixtho** [comment]
Another option could also be to have this enabled by default on votes like add time/skip and off by default on custom votes.
